service/src/main/java/uk/gov/hmcts/reform/fpl/model/docmosis/DocmosisGeneratedOrder.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.fpl.model.docmosis;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+import uk.gov.hmcts.reform.fpl.enums.EPOType;
+import uk.gov.hmcts.reform.fpl.enums.GeneratedOrderType;
+
+import java.util.List;
+
+@Getter
+@SuperBuilder
+@EqualsAndHashCode(callSuper = true)
+public class DocmosisGeneratedOrder extends DocmosisOrder {
+    private final String orderTitle;
+    private final String childrenAct;
+    private final String orderDetails;
+    private final String localAuthorityName;
+    private final String childrenDescription;
+    private final EPOType epoType;
+    private final String includePhrase;
+    private final String removalAddress;
+    private final String epoStartDateTime;
+    private final String epoEndDateTime;
+    private final GeneratedOrderType orderType;
+    private final String familyManCaseNumber;
+    private final String courtName;
+    private final String dateOfIssue;
+    private final String judgeTitleAndName;
+    private final String legalAdvisorName;
+    private final List<DocmosisChild> children;
+    private final Integer childrenCount;
```

Comment:
I don't see why this count field would be needed

Reply:
The original code that was using a map was populating this childrenCount field. If you say it is not needed, I will remove.

Reply:
I think can remove as it's not being used. [confirmation here](https://github.com/hmcts/fpl-ccd-configuration/pull/1175#discussion_r420725916)

Reply:
It turns out the childrenCount field is used in the docmosis template file - FL-PLW-GOR-ENG-00218.doc. So it is needed. Also I need to update the template to use judgeAndLegalAdvisor.

Reply:
children count can be removed and you can use docmosis template language instead. I remember a child count that was used in cmo that was no longer needed, so I would rather than change there

Reply:
Thanks for clearing that up. I will take a look.